tests/test_user_panel.py

```diff
@@ -0,0 +1,61 @@
+from typing import Union
+
+from django.contrib.auth import get_user
+from django.contrib.auth.models import AnonymousUser, User
+from django.http import HttpRequest
+from django.test import TestCase
+from django.urls import reverse
+
+
+class UserPanelTests(TestCase):
+    def assertCurrentUser(self, expected_user: Union[User, AnonymousUser]) -> None:
+        request = HttpRequest()
+        request.session = self.client.session
+        user = get_user(request)
+        self.assertEqual(expected_user, user, "Wrong user on session")
+
+    @classmethod
+    def setUpTestData(cls) -> None:
+        cls.user = User.objects.create(
+            email='demo@example.com',
+            username='demo-example',
+        )
+        User.objects.create(
+            email='other@example.com',
+            username='other-example',
+        )
```

Comment:
Might have missed something, but why is it that we create this additional user?

Reply:
Only to be sure that a trivial implementation of the behaviour wouldn't necessarily pass the tests.
As a general pattern I find it's useful to have have data available so that wrong behaviour could actually happen.